packages/dds/tree/src/test/rebase/generateFuzzyCombinedChange.spec.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { ChangeRebaser } from "../../rebase";
+import { AnchorSet } from "../../tree";
+import { generateFuzzyCombinedChange } from "./fuzz";
+
+const testSeed = 432167897;
+
+type TChange = TChange[] | { I: TChange; } | { C: TChange; O: TChange; } | string;
```

Comment:
The prefix "T" in `TChange` is typically used to convey that the type is a type parameter, which is not the case here. I think "Change" or "TestChange" would be fine.